examples/dynamic/views/Content.js

```diff
@@ -0,0 +1,43 @@
+var React = require('react');
+
+var TodoList = React.createClass({
+  render: function() {
+    var i = 0;
+    var createItem = function(itemText) {
+      return <li key={i++}>{itemText}</li>;
```

Comment:
This doesn't matter *at all* but you could also take advantage of the fact that `map` will call this with 3 args, the 2nd being the index :)